kube/src/api/subresource.rs

```diff
@@ -37,11 +35,9 @@ where
 
 // ----------------------------------------------------------------------------
 
-/// Status subresource
-///
-/// https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#status-subresource
-/// TODO: Replace examples with owned custom resources. Bad practice to write to owned objects
-/// These examples work, but the job controller will totally overwrite what we do.
```

Comment:
Removed `TODO:` and replaced the raw URL.
<img src="https://user-images.githubusercontent.com/639336/103121141-aa547100-462f-11eb-8485-d58b7517a889.png" width=600>

<img src="https://user-images.githubusercontent.com/639336/103121160-c7893f80-462f-11eb-8c2d-0bb46d092dca.png" width=600>